web-src/src/index.css

```
@@ -133,4 +133,57 @@ body {
 .variant-image-button {
   opacity: 0;
   transition: opacity 0.3s ease-in-out 0.1s;
-}
\ No newline at end of file
+}
+
+.slick-prev:before {
+  transform: rotate(180deg);
+}
+
+.slick-prev:before, .slick-next:before {
+  opacity: 1;
+  content: '';
+  width: 64px;
+  height: 64px;
+  display: block;
+  position: absolute;
+  top: 0;
+  background-size: contain;
+  background: url('./assets/variations-scroll-indicator.svg');
+}
+.slick-arrow {
+  background: #fff;
+  z-index: 2;
+  border-radius: 50%;
+  width: auto;
+  height: auto;
+  filter: opacity(0);
```

I found 0.2s transition time work best for the hovering effect.
```suggestion
  opacity: 0;
  transition: opacity 0.2s ease-in-out 0.1s;
```

---

fixed